Automattic/wp-calypso
# Editor: Reduxify The EditorTitle component

- Drop sitesList usage #8726 
- Drop PostEditStore calls
- Rewrite using the last coding style (ES6 class)

**Testing instructions**
- Create a new post and edit its title
- The title should update correctly
- Do the same thing for different post types

cc @aduth
